pkg/account/deployer/client.go

```diff
@@ -240,7 +240,7 @@ func (d *accountManagementClient) updateAccountPolicyBindings(ctx context.Contex
 	if groupId == "" {
 		return fmt.Errorf("group id must not be empty")
 	}
-	if len(policyIds) == 0 {
+	if policyIds == nil {
```

Comment:
I was thinking of treating policyIds == nil the same as policIds == empty first, but then decided to just do nothing.... no strong feeling.. wdyt?

Reply:
hmmm... can't think of an exact case where this would be called with an explicit nil list rather than an empty list. Did you think of one/thus handling them different actually make sense?
Gutfeeling: handling both as "remove bindings" would feel less prone to unintended behaviour in some edge cases we didn't think of yet

Reply:
yes, at the moment i fixed the code so that nil is never passed, but this could of course change in future when changing the code... will change to treat it like an empty list.